node/test/peer_strategies.js

```diff
@@ -207,6 +207,67 @@ allocCluster.test('prefer incoming selects the incoming peer when outgoing is pr
     }
 });
 
+allocCluster.test('subChannel: prefer incoming selects the incoming peer when outgoing is present' , {
+    numPeers: 3,
+    skipEmptyCheck: true,
+    preferConnectionDirection: 'out'
```

Comment:
If channelOptions is not provided, opts is used.
https://github.com/uber/tchannel/blob/master/node/test/lib/alloc-cluster.js#L58